hazelcast/src/test/java/com/hazelcast/nio/tcp/nonblocking/iobalancer/IOBalancerStressTest.java

```diff
@@ -74,7 +74,7 @@ public void testEachConnectionUseDifferentSelectorEventually() {
         IMap<Integer, Integer> map = instance1.getMap(randomMapName());
 
         long deadLine = System.currentTimeMillis() + TEST_DURATION_SECONDS * 1000;
-        for (int i = 0; System.currentTimeMillis() < deadLine; i++) {
+        for (int i = 0; System.currentTimeMillis() < deadLine && map.size() > 1000; i++) {
```

Comment:
Since the purpose here is to remove time sensitivity, why not just iterate `while (map.size() < 1000)` and disregard time?